site/content/concepts/command-and-control.md

```diff
@@ -66,3 +66,24 @@ The response of the command will be sent by the device to
 If the device is authenticated, the `tenant` and `device-id` are left empty (resulting in 3 subsequent `/`s).
 
 ![Command & Control over MQTT Adapter](../command_control_concept_mqtt.png) 
+
+## Command & Control over AMQP Adapter
+
+When an authenticated device connected to the AMQP adapter wants to receive commands from the adapter, it opens a receiver link at the source address:
+
+* `control`
+
+while unauthenticated devices have to fully specify their `tenant` and `device-id` as source address. 
```

Comment:
It would be better to show the source address here as well (instead of putting it into words), like
`control/tenant/device-id`.